Marlin/src/lcd/e3v2/proui/meshviewer.cpp

```diff
@@ -127,12 +133,11 @@ void MeshViewer::draw(const bool withsave/*=false*/, const bool redraw/*=true*/)
   else
     DWINUI::drawButton(BTN_Continue, 86, 305);
 
-  #if ENABLED(USE_GRID_MESHVIEWER)
-    bedLevelTools.setMeshViewerStatus();
-  #else
-    char str_1[6], str_2[6] = "";
-    ui.status_printf(0, F("Mesh minZ: %s, maxZ: %s"), dtostrf(min, 1, 2, str_1), dtostrf(max, 1, 2, str_2));
-  #endif
+  if (see_mesh) {
+    TERN_(USE_GRID_MESHVIEWER, bedLevelTools.setMeshViewerStatus());
+  }
+  else
+    ui.set_status_and_level(MString<30>(F("Mesh Z min: "), p_float_t(min, 2), F(", max: "), p_float_t(max, 2)));
```

Comment:
I propose we change this to 
```suggestion
    ui.set_status_and_level(MString<32>(
      F("Zmin: "), p_float_t(min, 3), F(" | "), p_float_t(max, 3), F("+"), F(" :Zmax"))
    );
```
it will look like
```suggestion
Zmin: -0.000 | 0.000+ :Zmax
```

of course we can always change it around
```suggestion
-0.000 :Zmin | Zmax: 0.000+
```